src/main/resources/config/application.yml

```diff
@@ -20,6 +19,8 @@ server:
     mime-types: application/json,application/xml,text/html,text/xml,text/plain
 
 revocation:
+  authorization:
+    reload: 600000
```

Comment:
@vroldanbet maybe we stay with milliseconds as @Scheduled annotation only supports that: http://docs.spring.io/spring/docs/4.1.x/javadoc-api/org/springframework/scheduling/annotation/Scheduled.html#fixedDelayString--

at least add "Milliseconds" to the property to make it clear